plutus-core/plutus-core/src/PlutusCore/Constant/Debug.hs

```diff
@@ -45,9 +43,10 @@ enumerateDebug = id
 -- >>> :t inferDebug False
 -- inferDebug False :: Bool
 -- >>> :t inferDebug $ Just 'a'
--- <interactive>:1:1-21: error:
---     • There's no 'KnownType' instance for Maybe Char
---       Did you add a new built-in type and forget to provide a 'KnownType' instance for it?
+-- <interactive>:1:1: error:
+--     • No instance for (KnownPolytype
+--                          (ToBinds (Maybe Char)) TermDebug '[] (Maybe Char) (Maybe Char))
+--         arising from a use of ‘inferDebug’
```

Comment:
I dropped that custom type error some time ago, 'cause it was playing badly with some other parts of the code. I'll try to reintroduce it later.